task_processing/plugins/smartstack.py

```diff
@@ -0,0 +1,34 @@
+import yaml
+
+
+YOCALHOST_IP = '169.254.255.254'
+TASKPROC_SERVICE_FILE = '/nail/etc/services/task_processing/smartstack.yaml'
```

Comment:
i don't think these values should be anywhere in our repo. we should make library users provide them when initializing `MesosExecutor`.